WORKFLOW.md

```diff
@@ -1,48 +1,62 @@
 # Development workflow
 
-Principles:
+## Introduction
 
-- Rely on automation as much as possible.
-- Automated tests should prevent bad deployments. Continuous integration,
-  continuous deployment.
+The HOPR Association team members have agreed on the following development workflow to streamline the process we use to implement the HOPR protocol.
 
-## Goal Workflow
+## Principles
 
-- A team-member, or an external contributor writes code in a feature branch, or
-  in their own fork.
+- **Automation-first**: Rely on automation as much as possible.
+- **Tests & CI**: Tests should prevent bad deployments and API regressions. Continuous integration ensures all our code is tested before being merged into our baseline.
+- **Releases & CD**: Every week, we do a code-freeze in our codebase by branching out a specific release, which we then deploy on every change.
 
-- The PR should track master
+## Rules
 
-- They create a draft pull-request while they work on the code so we can stay
-  up to date on progress.
+- All PR‘s should track to master.
+- All PR‘s have to be approved by two team members.
+- All PR‘s must pass all status checks before merging.
+- `master` tracks `release/**`, but releases do not track `master`. i.e. we always merge `release/**` back to `master`, but we do not merge `master` back to a frozen `release`.
```

Comment:
Master doesn't track anything.

Releases _can_ be merged back to master, but not always necessary.

Reply:
Gotcha, replaced the last one with `Releases can be merged back to master, but not always necessary.`